samples/sample_decode/src/pipeline_decode.cpp

```diff
@@ -345,14 +345,13 @@ mfxStatus CDecodingPipeline::Init(sInputParams *pParams)
         m_memType = pParams->memType;
     else
     {
+#if defined(_WIN32) || defined(_WIN64)
+        m_memType = pParams->bUseHWLib ? D3D11_MEMORY : SYSTEM_MEMORY;
+#elif defined(LIBVA_SUPPORT)
         switch (pParams->mode)
         {
         case MODE_PERFORMANCE:
-#if defined(_WIN32) || defined(_WIN64)
-            m_memType = pParams->bUseHWLib ? D3D11_MEMORY : SYSTEM_MEMORY;
-#elif defined(LIBVA_SUPPORT)
             m_memType = pParams->bUseHWLib ? D3D9_MEMORY : SYSTEM_MEMORY;
```

Comment:
Can you please create aliss in enum VAAPI_MEMORY = D3D9_MEMORY and use in if (LIBVA_SUPPORT) block?